### ISSUE TITLE
django session backend does not compatible with django 1.4.1alpha

### ISSUE THREAD

**Issue by observerss** [hmarr/mongoengine#402]
After upgraded to django 1.4.1alpha, session backend doesn't work

django raises an Exception says

---

**Comment by osantana** [comment]
The Django changeset 17155 (https://code.djangoproject.com/changeset/17155) make self.session_key read only. This change fix the issue 11555 (https://code.djangoproject.com/ticket/11555).

I've created a patch to mongoengine.django.sessions but I'm not sure if this is the right way to fix this issue but it's working here:

``` diff
--- sessions.py.orig    2012-01-12 12:14:14.000000000 -0200
+++ sessions.py 2012-01-12 12:14:23.000000000 -0200
@@ -35,7 +35,7 @@

     def create(self):
         while True:
-            self.session_key = self._get_new_session_key()
+            self._session_key = self._get_new_session_key()
             try:
                 self.save(must_create=True)
             except CreateError:
```

**Comment by rooty123** [comment]
Thanks for patch, it fixed also a "Can't set attribute" exception with django 1.4a

**Comment by rozza** [comment]
Fixed in 0.6.3
